lib/MusicBrainz/Server/WebService/JSONSerializer.pm

```diff
@@ -54,6 +54,8 @@ sub work_list          { shift->entity_list(@_, "work", "works") };
 sub area_list          { shift->entity_list(@_, "area", "areas") };
 sub place_list         { shift->entity_list(@_, "place", "places") };
 sub event_list         { shift->entity_list(@_, "event", "events") };
+sub instrument_list    { shift->entity_list(@_, "instrument", "instruments") };
+sub series_list        { shift->entity_list(@_, "series", "series") };
```

Comment:
Do you mean rewriting serialize() so that it no longer relies on a named sub lookup? I can look into that.
Or do you mean the plural form only? Looking at entities.json, plural_url looks like it has the value I need; is that guaranteed to be the value intended to be used in the JSON serialization?